src/Sylius/Behat/Context/Setup/CheckoutContext.php

```diff
@@ -96,6 +96,32 @@ public function iHaveProceededThroughCheckoutProcess(): void
         $this->completeCheckout($cart);
     }
 
+    /**
+     * @Given /^I have specified the billing (address as "([^"]+)", "([^"]+)", "([^"]+)", "([^"]+)" for "([^"]+)")$/
+     */
+    public function iHaveSpecifiedDefaultBillingAddressForName(): void
+    {
+        $cartToken = $this->sharedStorage->get('cart_token');
+
+        $command = new UpdateCart(null, $this->getDefaultAddress());
+        $command->setOrderTokenValue($cartToken);
+        $this->commandBus->dispatch($command);
+    }
+
+    /**
+     * @Given I proceeded with :shippingMethodName shipping method and :paymentMethodName payment method
+     */
+    public function iHaveProceededWithSelectingPaymentMethod(): void
```

Comment:
You are completing the checkout in this step without even choosing given shipping method and payment method, it looks that sth went wrong here 👀